### PR TITLE
Update 10-Kube-Proxy.md

### PR DESCRIPTION
**Pull request by adriancuadrado**
Fixed small typo

### CODE REVIEW

**Comment by adriancuadrado** [comment]
> `s/Yhis/This/`

Oof

**Comment by adriancuadrado** [comment]
> `s/Yhis/This/`

Done

**Comment by stappersg** [comment]
On Wed, Dec 04, 2024 at 02:05:22PM -0800, adriancuadrado wrote:
> > `s/Yhis/This/`
> 
> Done

Thanks.

Now we wait (again)


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Review by stappersg — APPROVED** [review]
Please merge, so we can move on. (Please merge, so mankind doesn't loose further human energy on _"Why is this sane merge request still not merged?"_)

**Comment by adriancuadrado** [comment]
@stappersg I don't have write access to the repository so I can't merge.

**Comment by stappersg** [comment]
```text
> > Please merge, so we can move on.
> I don't have write access to the repository so I can't merge.
```
@adriancuadrado  just leave it to those who can.  And it is okay to move on.
